game.py

```diff
@@ -0,0 +1,150 @@
+import random
+import string
+
+
+class Game:
+    def __init__(self, size=3, target=100, level='medium'):
```

Comment:
> nit. board_size?

changed it to board_size, much clearer now!



> meaning of "target"?

target means the target for the score you have to reach to win the game. I renamed it to score_target_to_win, so it is clear and took it out of the init-method because it is not really necessary there. by setting the level to easy, medium, or hard it will be set automatically now.